graphql_compiler/compiler/sqlalchemy_extensions.py

```diff
@@ -64,15 +66,31 @@ def print_sqlalchemy_query_string(query, dialect):
     Returns:
         string that can be ran using sqlalchemy.sql.text(result)
     """
+    supported_dialects = (PGDialect_psycopg2, MSDialect_pyodbc)
+    if isinstance(dialect, supported_dialects):
+        class BindparamCompiler(dialect.statement_compiler):
```

Comment:
Since `dialect` here is of a dynamically-specified type, would you mind type-hinting this file (perhaps in a separate PR that merges first) and then adding type hints to all these functions? It's possible there's no other way to do this, but I'd rather hit mypy issues when this code is in your brain's cache than sometime down the road when nobody remembers how and why this works :)